ceph-releases/jewel/ubuntu/16.04/demo/Dockerfile

```diff
@@ -7,6 +7,13 @@
 FROM ceph/base:tag-build-master-jewel-ubuntu-16.04
 MAINTAINER Sébastien Han "seb@redhat.com"
 
+# Install s3cmd to support creation of buckets
+RUN ["apt-get", "update", "-y"]
+RUN ["apt-get", "install", "-y", "python-dateutil", "python-magic"]
+
+RUN ["wget", "http://mirrors.kernel.org/ubuntu/pool/universe/s/s3cmd/s3cmd_1.6.1-1_all.deb"]
+RUN ["dpkg", "-i", "s3cmd_1.6.1-1_all.deb"]
```

Comment:
I meant there is no need to fetch the `.deb`, can't we just install s3cmd from the Ubuntu package source?